### PR TITLE
Add google-cloud-monitoring gem

### PR DESCRIPTION
**Pull request by natarajaya**
Related to https://github.com/gpii-ops/gpii-infra/pull/183.

### CODE REVIEW

**Comment by mrtyler** [comment]
LGTM

Oof, g++ is going to inflate our image size.

File: dockerfiles/google/Dockerfile [hunk 1]
```diff
@@ -43,10 +43,13 @@ RUN apk --no-cache add \
         apache2-utils \
         tzdata \
         jq \
+        make \
+        g++ \
```
Comment (stepanstipl) [inline dockerfiles/google/Dockerfile]: We should add this, then install the gem, and then remove this in one step -> that way image stays small. Imho not leaving make and other build bits in the final image is much nicer.

Comment (natarajaya) [reply dockerfiles/google/Dockerfile]: @stepanstipl Thanks for the suggestion, I had the same idea!

**Comment by mrtyler** [comment]
LGTM

**Comment by stepanstipl** [comment]
+1 yep, thanks @natarajaya, this looks much better to me

**Pull request merged**